### ISSUE TITLE
HELP: i cant understand the way to add Zigbee devices behind a Zigbee gateway. PLEASE HELP

### ISSUE THREAD

**Issue by jhony251** [rospogrigio/localtuya#1499]
could anyone help me to learn how to configure a new zigbee device.

PLEASE .

**Comment by jwinut** [comment]
I use z2m. in z2m, allow new devices to join. to make device join, read device's manual and make it to pairing mode. for most devices, hold down power button 5-10 secs or locate rst/reset/pair button and use something to poke inside.
![Screenshot 2566-09-22 at 14 49 55](https://github.com/rospogrigio/localtuya/assets/8947455/6d48976d-e08b-4cf9-a248-791a781edd75)

**Comment by jhony251** [comment]
yes me too.  i usually work with Z2M, but i would like to be able to make work this addon with ZB because there are some devices that dont send all info in right way throw Z2M

**Comment by nicknol** [comment]
zigbee is not a topic for localTuya. localTuya is about Tuya devices connected via Wifi

**Comment by jhony251** [comment]
Thanks Everyone. 
I  have  got work right. Was a problem about uderstand each parametter

best regards.

**Comment by yabdali** [comment]
> 

How did you add Zigbee devices behind a Zigbee gateway? Is the Zigbee Gateway Tuya brand or something else like a USB stick/dongle? I have ordered some devices to test and among them is Tuya Gateway (Multi-mode by MOES). So far I haven't been able to find clear directions whether I would be able to control Zigbee devices behind Tuya Gateway. Can you please share your experience? I am intending to run Home Assistant to automate my smart home devices but using Tuya Gateway.

**Comment by joaocbatista** [comment]
from what I could understand, currently, the only possible way is to have a usb dongle gateway that you add to your homeassistant machine and have the devices communicating directly with that gateway.

From the fork I've shared above, it seems the developer was able to add support using a tuya wifi gateway, but that's not included in the main fork
